recipe/0001-Add-second-downstream-patch.patch

```diff
@@ -0,0 +1,26 @@
+From 636e7dcfe2e23d9cea8f260d151e6b04bc8c4eb8 Mon Sep 17 00:00:00 2001
+From: BastianZim <10774221+BastianZim@users.noreply.github.com>
+Date: Sun, 3 Oct 2021 14:50:44 +0200
+Subject: [PATCH] Add second downstream patch
+
+---
+ Clp/src/OsiClp/osi-clp.pc.in | 5 +++--
+ 1 file changed, 3 insertions(+), 2 deletions(-)
+
+diff --git a/Clp/src/OsiClp/osi-clp.pc.in b/Clp/src/OsiClp/osi-clp.pc.in
+index aeb04e93..af1114f7 100644
+--- a/Clp/src/OsiClp/osi-clp.pc.in
++++ b/Clp/src/OsiClp/osi-clp.pc.in
+@@ -7,6 +7,7 @@ Name: OsiClp
+ Description: COIN-OR Open Solver Interface for CLP
+ URL: https://projects.coin-or.org/Osi
+ Version: @PACKAGE_VERSION@
+-Libs: -L${libdir} -lOsiClp @OSICLPLIB_PCLIBS@
++Libs: -L${libdir} -lOsiClp
++Libs.private: @OSICLPLIB_PCLIBS@
+ Cflags: -I${includedir}
+-Requires: clp @OSICLPLIB_PCREQUIRES@
++Requires.private: clp @OSICLPLIB_PCREQUIRES@
```

Comment:
I think this Requires was too much :) Should not be private. @tkralphs @BastianZim

Reply:
Ok, will open a PR and then I'd suggest waiting for @tkralphs feedback as he recommended adding it - I cannot say whether this is required or not. :)

Reply:
@wolfv Would you mind creating a 1.17.4 branch because we'll need to move the changes back to there again and I'm not sure if it's the last time we'll have to do so... :) I'll make the PR against that one then.

Reply:
:+1:

Reply:
made the branch

Reply:
Thanks!

Reply:
PRs are created but https://github.com/conda-forge/coin-or-cbc-feedstock/pull/11#issuecomment-959005823 disagrees.